src/components/SocialButtonsList/SocialButtonsList.tsx

```diff
@@ -4,10 +4,13 @@ import SocialButton, { SocialButtonVariant } from '../SocialButton/SocialButton'
 
 import styles from './SocialButtonsList.module.scss';
 
-import { getSocialLoginUrls } from '#src/stores/AccountController';
+import AccountController from '#src/stores/AccountController';
+import { getModule } from '#src/modules/container';
 
 const SocialButtonsList = () => {
-  const urls = useQuery('socialUrls', getSocialLoginUrls);
+  const accountController = getModule(AccountController);
+
+  const urls = useQuery('socialUrls', () => accountController.getSocialLoginUrls());
```

Comment:
This one is really tricky :) We lose the context (access to `this`) this way in case we have a traditional class method (instead of an arrow function). Do we want to rewrite all classes to use arrow functions instead of classic methods?

Reply:
I don't understand. Which `this` are you referring to? Inside the getSocialLoginUrls method?

Reply:
Right. `this` we use to refer to the `accountService` or `checkoutService`.